### PR TITLE
BRITECHSRM-271 Add date and dateAdded

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

**Comment by ianlovell** [comment]
:shipit:

File: Sessions/api.json [hunk 1]
```diff
@@ -119,47 +119,13 @@
...
-                    "description": "The unique identifier of the administrator for the session"
-                }
-            }
-        },
-        "SessionDetail": {
+        "Session": {
             "type": "object",
             "required": [
                 "id",
-                "title"
```
Comment (dwybourn) [inline Sessions/api.json]: Did you mean to remove title? The description on line 23 says we return it.

Comment (lclarksl) [inline Sessions/api.json]: We still return it - this just means that it could be empty i.e. we have a speaker but not a talk name yet

**Comment by dwybourn** [comment]
:shipit:

**Pull request merged**